jupyterhub/user.py

```diff
@@ -223,6 +223,18 @@ def authenticator(self):
     def spawner_class(self):
         return self.settings.get('spawner_class', LocalProcessSpawner)
 
+    async def sync_groups(self, user_groups):
```

Comment:
```suggestion
   def sync_groups(self, user_groups):
```

this isn't async. Add a docstring describing what it does?